packages/common/src/utils/xShare.ts

```diff
@@ -1,3 +1,10 @@
+import type { User } from '../models'
+
+export const getXShareHandle = (user: User) => {
+  const xHandle = user.twitter_handle
+  return xHandle ?? user.handle
```

Comment:
this seems wrong, it will tag the X user that has the audius users audius handle

Reply:
oic good point thx